### PR TITLE
Winston updates for better logging

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Proposed changes

Winston updates for better logging
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
## Further comments

@imurchie Few updates,
- https://github.com/appium/appium/blob/master/lib/logsink.js#L146 Still persist in latest version so i left it as it is with unlink
- https://github.com/appium/appium/blob/master/lib/logsink.js#L199 Ah, yes winston has a streaming API now but help me in implementing this, https://github.com/winstonjs/winston#streaming-logs.

### CODE REVIEW

**Comment by moizjv** [comment]
Can one of the admins verify this patch?

**Comment by SrinivasanTarget** [comment]
https://github.com/appium/appium/issues/6286

File: lib/logsink.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ function _createWebhookTransport (args, logLvl) {
     port = parseInt(hostAndPort[1], 10);
   }
 
-  let transport = new (winston.transports.Webhook)({
+  let transport = new (winston.transports.Http)({
     name: "webhook",
```
Comment (bootstraponline) [inline lib/logsink.js]: If the transport is http, why is the name webhook?

**Comment by SrinivasanTarget** [comment]
@imurchie @bootstraponline ping

**Comment by SrinivasanTarget** [comment]
Resolved conflicts now

**Comment by imurchie** [comment]
Looks good.

**Pull request merged**